package.json

```diff
@@ -55,11 +55,13 @@
   ],
   "dependencies": {
     "@babel/polyfill": "^7.8.3",
-    "@datapunt/asc-assets": "^0.18.1",
-    "@datapunt/asc-core": "^0.18.1",
-    "@datapunt/asc-ui": "^0.18.2",
+    "@datapunt/amsterdam-react-maps": "git+https://github.com/Amsterdam/amsterdam-react-maps.git#feature/package-export",
+    "@datapunt/asc-assets": "0.18.2-alpha.29",
```

Comment:
This is a canary version and shouldn't be used in production. Why would we need this version?

Reply:
There are some breaking changes in the amsterdam-react-maps that work with this version, will replace it asa theese changes are implemented in the main version